firestore/firestore_snippets/query.go

```diff
@@ -259,6 +269,16 @@ func createArrayContainsQuery(ctx context.Context, client *firestore.Client) err
 	return nil
 }
 
+func createArrayContainsAnyQuery(ctx context.Context, client *firestore.Client) error {
+	cities := client.Collection("cities")
+	// [START fs_array_contains_any_query]
```

Comment:
```suggestion
	// [START fs_query_filter_array_contains_any]
```

Reply:
Can you move the start tag above the `cities := ...` line, we want that in the snippet.

Reply:
The array-contains snippet in this file does not include that, but happy to include it in these.